src/plugins/data_table/dataTables.colReorder.js

```diff
@@ -519,8 +527,10 @@
     ColReorder.aoInstances.push(this);
 
 
+    var tableElement = $(this.s.dt.nTable);
+
     if (this.s.bResizeTableWrapper) {
-      $(this.s.dt.nTableWrapper).width($(this.s.dt.nTable).width());
+      tableElement.width(tableElement.width());
```

Comment:
and setting it's width... to it's width?  what does this do, and if it's on purpose can you add a comment please?